mscore/palette/palettelistview.cpp

```diff
@@ -0,0 +1,108 @@
+//=============================================================================
+//  PaletteListView
+//
+//  Copyright (C) 2020 Peter Jonas
```

Comment:
Not MuseScore BVBA?

Reply:
Nope. I wrote the code so it is my copyright. MuseScore has permission to use it under the terms of the CLA, and everybody else under GPL version 2.

Reply:
OK. Or soon under GPL version 3 I guess

Reply:
Right, but they will always be able to use this particular file under GPL 2 if they prefer. The CLA gives MuseScore the right to offer the file under other licenses *in addition to GPL 2*. The project as a whole may become GPL 3, but this file would be dual-licensed GPL 2 + GPL 3.